flutter/news/lib/core/error/failures.dart

```diff
@@ -5,4 +5,16 @@ abstract class Failure extends Equatable {
 
   @override
   List<Object> get props => const [];
+}
+
+class ServerFailure extends Failure {
+  final String message;
+
+  const ServerFailure(this.message);
+}
+
+class CacheFailure extends Failure {
+  final String message;
+
+  const CacheFailure(this.message);
 }
```

Comment:
same, `no newline at the end of the file`

Reply:
updated on https://github.com/novoda/multiplatform-playground/commit/8323a52c3c021a85d1662fa2609c061534a5fce3